### DISCUSSION TITLE
Brands: Google, Facebook, GitHub, Twitter, Instagram, Apple, etc.

### DISCUSSION THREAD

**Discussion by dorianmariecom** [tailwindlabs/heroicons#599] [Ideas]
what do you think? I can contribute them, and make them myself in inkscape

Also, would be even better to have GMail, Google Calendar, etc. for Google

here is instagram for instance https://github.com/dorianmariefr/heroicon/blob/master/app/assets/images/heroicon/outline/instagram.svg

**Comment by zefir-git** [comment]
Why in Heroicons? Especially since 3000+ brand icons are already actively maintained by https://simpleicons.org/?
